config/samples/usecases/move-all-clusters/move-all-managed-clusters.yaml

```diff
@@ -1,30 +1,30 @@
+# - you want to move ALL managed clusters to Hub2
+# - Hub2 manages some other clusters 
 ########################################
 
 
-# Before applying the move-all-managed-clusters resource on hub-b:
-# 1. Pause BackupSchedule on hub-a
-# 2. After BackupSchedule is set to paused on hub-a, set this label annotation on ALL ManagedCluster ( except local cluster )
-#    on hub-a :  import.open-cluster-management.io/disable-auto-import: ''
-#    This annotation will prevent hub-a trying to recover any of the managed clusters after being moved to hub-b
+# Before applying the move-all-managed-clusters resource on Hub2:
+# 1. Pause BackupSchedule on Hub1
+# 2. After BackupSchedule is set to paused on Hub1, set this label annotation on ALL ManagedCluster ( except local cluster )
+#    on Hub1 :  import.open-cluster-management.io/disable-auto-import: ''
+#    This annotation will prevent Hub1 trying to recover any of the managed clusters after being moved to Hub2
 
-# 3. Pause any BackupSchedule on hub-b
-# 4. Update DateProtectionApplication resource on hub-b to point to the same storage location as hub-a
-# 5. Wait until you see the backups from hub-a being synced up on hub-b
+# 3. Pause any BackupSchedule on Hub2
+# 4. Update DateProtectionApplication resource on Hub2 to point to the same storage location as Hub1
+# 5. Wait until you see the backups from Hub1 being synced up on Hub2
 
-# After the move-all-managed-clusters restore is applied on hub-b and has a status of Finished, wait and look for a status Ready 
-# for each managed cluster moved from hub-a to hub-b. When this happens, the cluster is managed by hub-b.
-# To remove the managed clusters from hub-a, for each managaed cluster, wait until the status is Unknown on hub-a. Only when 
-#  this happens, delete the ManagedCluster global resource on hub-a. This will clean up the
-#  managed cluster resources on hub-a but since the cluster status it Unknown, it will not run any detach operation - which you don't want to happen.
+# After the move-all-managed-clusters restore is applied on Hub2 and has a status of Finished, wait and look for a status Ready 
+# for each managed cluster moved from Hub1 to Hub2. When this happens, the cluster is managed by Hub2.
+# To remove the managed clusters from Hub1, for each managaed cluster, wait until the status is Unknown on Hub1. Only when 
+#  this happens, delete the ManagedCluster global resource on Hub1. This will clean up the
+#  managed cluster resources on Hub1 but since the cluster status it Unknown, it will not run any detach operation - which you don't want to happen.
 
-# You can now resume the BackupSchedules on hub-a.
-# Update the DateProtectionApplication on hub-b to point to hub-b storage location. You can resume now the BackupSchedule on hub-b.
+# You can now resume the BackupSchedules on Hub1.
+# Update the DateProtectionApplication on Hub2 to point to Hub2 storage location. You can resume now the BackupSchedule on Hub2.
```

Comment:
Didn't notice this before, typo `DateProtectionApplication` should be `DataProtectionApplication`?

Reply:
fixed! thanks